scripts/run_node.sh

```diff
@@ -91,9 +12,15 @@ then
     echo "If you wish to customize the directory, select 'n' and re-run the script"
     echo "with the '--data_dir' argument."
     echo "Do you want to continue? [Y/n]"
-    read -r CONT
+    
+    if [[ "$PROMPTS" = true ]]
+    then
+        read -r CONT
```

Comment:
Not exactly issue of this PR, but maybe worth "fixing":

I see that in some places we are reading (potentially) multiple characters for y/n prompts, while in other we read just one (`-n 1` read option). Both options are fine, but I think we should be consistent.